node/pkg/watchers/sui/watcher.go

```diff
@@ -155,30 +147,32 @@ var (
 // NewWatcher creates a new Sui appid watcher
 func NewWatcher(
 	suiRPC string,
-	suiWS string,
 	suiMoveEventType string,
 	unsafeDevMode bool,
 	messageEvents chan<- *common.MessagePublication,
 	obsvReqC <-chan *gossipv1.ObservationRequest,
 ) *Watcher {
 	return &Watcher{
-		suiRPC:           suiRPC,
-		suiWS:            suiWS,
-		suiMoveEventType: suiMoveEventType,
-		unsafeDevMode:    unsafeDevMode,
-		msgChan:          messageEvents,
-		obsvReqC:         obsvReqC,
-		readinessSync:    common.MustConvertChainIdToReadinessSyncing(vaa.ChainIDSui),
-		subId:            0,
+		suiRPC:                    suiRPC,
+		suiMoveEventType:          suiMoveEventType,
+		unsafeDevMode:             unsafeDevMode,
+		msgChan:                   messageEvents,
+		obsvReqC:                  obsvReqC,
+		readinessSync:             common.MustConvertChainIdToReadinessSyncing(vaa.ChainIDSui),
+		subId:                     0,
+		latestProcessedCheckpoint: 0,
+		maximumBatchSize:          10,
+		descendingOrder:           true, // Retrieve newest events first
+		loopDelayInSecs:           3,    // SUI produces a checkpoint every ~3 seconds
```

Comment:
Perhaps this could go a little faster, like every second, so we don't introduce an up-to 3 second delay?